File: main/java/nullbots/nullmage/CurseBear.java
package nullbots.nullmage;

import com.runemate.game.api.hybrid.region.Npcs;
import com.runemate.game.api.script.framework.tree.LeafTask;

public class CurseBear extends LeafTask {
    @Override
    public void execute() {
        Npcs.newQuery().names("Grizzly bear").results().nearest().click();
    }
}

File: kotlin/archbots/archmage/treebot/leaves/CurseBear.kt
package archbots.archmage.treebot.leaves

import com.runemate.game.api.osrs.local.hud.interfaces.Magic
import archapi.treebot.framework.LeafTask

class CurseBear : LeafTask() {
    override fun execute() {
//        Npcs.newQuery().names("Grizzly bear").results().nearest().click();
//        Magic.CAMELOT_TELEPORT.activate()
        Magic.WIND_STRIKE.activate()
    }
}
